external/settings.php

```diff
@@ -7,6 +7,12 @@
 
 $tmpl = new OCP\Template( 'external', 'settings');
 
-$tmpl->assign('images', glob(\OC_App::getAppPath('external') . '/img/*.*'));
+$images = glob(\OC_App::getAppPath('external') . '/img/*.*');
+$theme = \OC_Util::getTheme();
```

Comment:
Technically those two usages are private API and app developers should not use these.

@DeepDiver1975 Should we add those to the public API somehow?